### ISSUE TITLE
Restrict actions of admins on a space that is not approved

### ISSUE THREAD

**Issue by mconf-daileon** [mconf/mconf-web#771]
---

Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Redmine Issue: 1753, http://dev.mconf.org/redmine/issues/1753
Original Date: 2015-09-23
Original Assignee: Lucas Zawacki

---

When a space is created and the moderation of spaces is on, the space will be set as "not approved", but the user that created the space will be able to access it. This is good, so the user knows that the space was created and is able to adjust it if needed (e.g. change the name of the space).

However, some things should not be available until the space is approved. They are:
* The owner of the space should not be able to invite people to join the space;
* Other users should not be able to request to join it;
* The web conference room should be completely inaccessible: meetings cannot be created nor joined, recordings and meeting objects (@BigbluebuttonMeeting) should not be accessible at all;
* Posts and attachments should be disabled.
* In short, any other resource that is not related to the administration of the space should also be disabled.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2015-09-23T18:55:02Z

---

Issue #1771 has been updated by Lucas Zawacki.

Status changed from Queued to Feedback
Assignee changed from Lucas Zawacki to Leonardo Daronco
% Done changed from 0 to 90

https://github.com/mconf/mconf-web/pull/136
